### ISSUE TITLE
why mxnet is used over pytorch?

### ISSUE THREAD

**Issue by xiaoouwang** [emorynlp/bert-2019#2]
Hello dear authors:

Just out of curiosity, I'd like to know if there are some particular reasons on the use of mxnet over pytorch, since the latter is much more common in academic research :) If you don't mind, I'd like to fork the repository and propose an implementation in pytorch.

Best

**Comment by hankcs** [comment]
Hi there,

Thank you for your great question. This project was initiated in the late 2018 when PyTorch didn't completely dominate the reseach community and there was no Huggingface Transformers. Besides, we gratefully had internal support from the Gluon developer team so we went with MXNet, which is also a decent framework even in these days. It just took us quite a long time to get the paper published though. So you might feel it's using a framework that's less popular today.

Sure, I'm glad to see a PyTorch implementation of it. We also launched a PyTorch project called [ELIT](https://github.com/emorynlp/elit) that finetunes transformers instead of using them as static embeddings. Hope you find them helpful in your research.
